source/publish_project/version/index.html.md.erb

```diff
@@ -0,0 +1,14 @@
+---
+title: Version control
+weight: 72
+last_reviewed_on: 2019-10-03
+review_in: 1 day
+---
+
+# Use version control
+
+You should use version control when managing your site. For example, GDS use Git and GitHub.
+
+Refer to the [Technical Documentation Template ruby gem repo dependents page][tech-docs-gem-dependents] to see which repos currently use the template.
```

Comment:
```suggestion
You can see [which repos currently use the Tech Docs Template](tech-docs-gem-dependents) in the Tech Docs Template Ruby gem GitHub repo.
```

Reply:
Could you do a quick check that we've been consistent about using either 'Tech Docs Template' or 'Technical Documentation Template' throughout?

Reply:
Sorry, just spotted that this sentence is also on the 'Support' page, so I think it needs removing in one place or the other?